tool/src/main/java/org/wildfly/security/tool/FileSystemEncryptRealmCommand.java

```diff
@@ -0,0 +1,761 @@
+            this.secretKeyAlias = secretKeyAlias;
+        }
+
+        Boolean getPopulate() {
+            return this.populate;
+        }
+
+        void setPopulate(Boolean populate) {
+            this.populate = populate;
+        }
+
+        void reset() {
+            this.inputRealmLocation = null;
+            this.outputRealmLocation = null;
+            this.FileSystemRealmName = null;
+            this.credentialStore = null;
+            this.createCredentialStore = null;
+            this.secretKeyAlias = null;
+            this.hashEncoding = null;
+            this.encoded = null;
+            this.levels = 0;
+            this.populate = null;
+        }
+    }
+
+    @Override
+    public void execute(String[] args) throws Exception {
+        setStatus(GENERAL_CONFIGURATION_ERROR);
+        cmdLine = parser.parse(options, args, false);
+        setEnableDebug(cmdLine.hasOption(DEBUG_PARAM));
+        if (cmdLine.hasOption(HELP_PARAM)) {
+            help();
+            setStatus(ElytronTool.ElytronToolExitStatus_OK);
+            return;
+        }
+        if (cmdLine.hasOption(SILENT_PARAM)) {
+            silentMode = true;
+        }
+        if (cmdLine.hasOption(SUMMARY_PARAM)) {
+            summaryMode = true;
+            summaryString = new StringBuilder();
+            summaryString.append(String.join("", Collections.nCopies(SUMMARY_WIDTH, "-")));
+            summaryString.append(System.getProperty("line.separator"));
+            summaryString.append("Summary for execution of Elytron-Tool command FileSystemRealmEncrypt");
+            summaryString.append(System.getProperty("line.separator"));
+            summaryString.append(String.join("", Collections.nCopies(SUMMARY_WIDTH, "-")));
+            summaryString.append(System.getProperty("line.separator"));
+        }
+        printDuplicatesWarning(cmdLine);
+
+        String realmNameOption = cmdLine.getOptionValue("r");
+        String inputRealmLocationOption = cmdLine.getOptionValue("i");
+        String outputRealmLocationOption = cmdLine.getOptionValue("o");
+        String credentialStoreOption = cmdLine.getOptionValue("c");
+        String createCredentialStore = cmdLine.getOptionValue("a");
+        String secretKeyAliasOption = cmdLine.getOptionValue("s");
+        String hashEncodingOption = cmdLine.getOptionValue("e");
+        String levelsOption = cmdLine.getOptionValue("l");
+        String encodedOption = cmdLine.getOptionValue("f");
+        String bulkConvert = cmdLine.getOptionValue("b");
+        String populateOption = cmdLine.getOptionValue("p");
+
+        if (bulkConvert == null) {
+            if (realmNameOption == null) {
+                realmNameOption = DEFAULT_FILESYSTEM_REALM_NAME;
+            }
+
+            Descriptor descriptor = new Descriptor();
+            descriptor.setFileSystemRealmName(realmNameOption);
+            descriptor.setOutputRealmLocation(Paths.get(outputRealmLocationOption).toString());
```

Comment:
When `outputRealmLocationOption` is not set `NullPointerException` is thrown

```
$ ./bin/elytron-tool.sh filesystem-realm-encrypt -i a --debug
Exception encountered executing the command:
java.lang.NullPointerException
	at java.base/sun.nio.fs.UnixPath.normalizeAndCheck(UnixPath.java:75)
	at java.base/sun.nio.fs.UnixPath.<init>(UnixPath.java:69)
	at java.base/sun.nio.fs.UnixFileSystem.getPath(UnixFileSystem.java:279)
	at java.base/java.nio.file.Path.of(Path.java:147)
	at java.base/java.nio.file.Paths.get(Paths.java:69)
	at org.wildfly.security.elytron-tool@1.18.4.CR1-SNAPSHOT//org.wildfly.security.tool.FileSystemEncryptRealmCommand.execute(FileSystemEncryptRealmCommand.java:325)
	at org.wildfly.security.elytron-tool@1.18.4.CR1-SNAPSHOT//org.wildfly.security.tool.ElytronTool.main(ElytronTool.java:85)
	at org.jboss.modules.Module.run(Module.java:353)
	at org.jboss.modules.Module.run(Module.java:321)
	at org.jboss.modules.Main.main(Main.java:604)
```

Reply:
Changed to
```
Exception encountered executing the command:
org.apache.commons.cli.MissingArgumentException: Output Realm location not specified.
	at org.wildfly.security.tool.FileSystemEncryptRealmCommand.execute(FileSystemEncryptRealmCommand.java:326)
	at org.wildfly.security.tool.ElytronTool.main(ElytronTool.java:85)
```